appcues/src/main/java/com/appcues/ui/composables/AppcuesComposition.kt

```diff
@@ -50,6 +49,7 @@ internal fun AppcuesComposition(
             LocalAppcuesActionDelegate provides DefaultAppcuesActionsDelegate(viewModel),
             LocalAppcuesPaginationDelegate provides AppcuesPagination { viewModel.onPageChanged(it) },
             LocalAppcuesTraitExceptionHandler provides AppcuesTraitExceptionHandler { viewModel.onTraitException(it) },
+            LocalAppcuesOverlayVisibility provides AppcuesOverlayVisibility { viewModel.updateOverlayVisibility(it) },
```

Comment:
we have several things now using the `viewModel` in a sort of abstracted way, for bits and pieces of functionality. We also have the `LocalViewModel` with the entire object available. It seems a bit redundant now to have all these other composition locals that wrap functions of the view model?